README.md

```diff
@@ -17,20 +17,30 @@ Welcome to the the Volunteering App Github repo
 4. Make sure that you have Android 10 installed and not higher.
 
 
-# Setup     
+# Setup  
 
 1. Ensure that you've gone through the following link for your particular platform: https://reactnative.dev/docs/environment-setup
 
 2. Pull the code from Git
  
-3. Open Command terminal       
- 
-4. Go to the `app` folder inside the project folder (e.g. **\Volunteer-app path\app**)      
+3. Open Command terminal
+
+4. Go to the `api` folder inside the project folder (e.g. **\Volunteer-app path\api**)
+
+5. Create .env file to store Jira API access credentials.
+    >**Note** Credentials themselves not provided, these should be requested/provided on joining the dev group.
+
+5. At the command prompt type `npm install` then `npm start` to start the Volunteer App API server.
```

Comment:
I think this should be `6.` ?  And some of the other numbers below need to be updated so they're up to date.  (It looks ok when I preview the README, but the numbering in the code should still be consistent.)

Reply:
Both points 5 are already in the readme file. 
Great suggestion to add the note that credentials can be obtained by joining the dev group.

Reply:
Renumbered - I didn't know Markdown ignored duplicate list numbers!